shopfloor/models/res_company.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2024 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    shopfloor_excluded_product_packaging_ids = fields.Many2many(
```

Comment:
Why not on the shopfloor app?

Reply:
I thought of it, but it seemed to me like a very specific setting compared to the rest of the settings in the shopfloor app (i.e. this doesn't really impact the app as a whole, but a specific component in shopfloor).

I can change it if you prefer.

Reply:
Thinking again on this: wouldn't make more sense to have a flag on pkg types?
CC @jbaudoux

Reply:
I think I would've added a flag at packaging to determine if the package is a vendor package or not.
This way we can filter this depending on the scenario / menu.
T'would probably make it easier to do smth like that `_product_packaging(rec, ..., show_vendor_pkgs=False)`.
show_vendor_pkgs taking the value of `menu.show_vendor_pkgs`.

Reply:
I've created a new module to enable this option here:
https://github.com/OCA/product-attribute/pull/1523

Reply:
I have now updated this PR to use the option in the PR in my last comment above.

Now the vendor packagings will not be displayed by default, and I enabled it in reception as discussed.

Reply:
I believe reception is an example, we need to determine where we want to see those packaging.